staging/src/k8s.io/api/autoscaling/v1/types.go

```diff
@@ -223,6 +232,14 @@ type PodsMetricSource struct {
 	// targetAverageValue is the target value of the average of the
 	// metric across all relevant pods (as a quantity)
 	TargetAverageValue resource.Quantity `json:"targetAverageValue" protobuf:"bytes,2,name=targetAverageValue"`
+
+	// selector is the selector for the given metric
```

Comment:
These need to be sentences - newlines are not considered significant and will cause this to be run on output.

Reply:
Done